Source/Evolve/WarpXEvolve.cpp

```diff
@@ -217,19 +225,18 @@ WarpX::Evolve (int numsteps)
                       << " s; This step = " << walltime_end_step-walltime_beg_step
                       << " s; Avg. per step = " << walltime/(step+1) << " s\n";
 
-        // sync up time
-        for (int i = 0; i <= max_level; ++i) {
-            t_new[i] = cur_time;
-        }
+        //// sync up time
```

Comment:
Do we still need this?

Reply:
I kept this one to be consistent with development branch and removed the previous one since it was added to compare the old and new BTD implementation.